src/store/services/transform-data.comparison.ts

```diff
@@ -0,0 +1,41 @@
+import { PetProfile } from '../../shared/interfaces/pet-profile.interface';
+import {  PetsToCompareList } from '../../pages/comparison/interfaces/pet.interface';
+
+export default function transformData(data: PetProfile[]) {
+    let result = [];
+  
+    for (let el of Object.keys(data[0].characteristics)) {
```

Comment:
I'd like to suggest creating a variable for data[0].characteristics, so it will make your code a little bit more self-explanatory

Reply:
Agree